# Total planning attempts
max_solutions: 6

# Planning group and link names
arm_group_name: "arm_with_torso"
eef_name: "gripper"
hand_group_name: "gripper"
hand_frame: "gripper_link"

# Poses
hand_open_pose: "open"
hand_close_pose: "close"
arm_home_pose: "ready"

# Scene frames
world_frame: "world"
base_frame: "base_link"
surface_link: "table"
surface_link_two: "shelf"

# 0: horizontal_frame_transform, 1: vertical_frame_transform, 2: diagonal_frame_transform
transform_orientation: 2

# Place offset from surface
place_surface_offset:  .001 

# Valid distance range when approaching an object for picking
approach_object_min_dist: 0.167
approach_object_max_dist: 0.2

# Valid height range when lifting an object after pick
lift_object_min_dist: 0.02
lift_object_max_dist: 0.1

object_offset: [.05, 0, 0, 0, 0, 0]

# BOX object specifications
pick_object: "Box 2"
# place_pose: [1.0280001163482666, 0.28, 0.726599931716919, 0, 0, 0]
place_pose:  [1, 0.0, 0.8, 0, 0, 0]